content/en/Getting started/track-credits.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Track Your Credits"
+linkTitle: "Track Your Credits"
+weight: 380
+description: >
+  View and track your credits in the dashboard.
+---
+
+{{% pageinfo %}}
+Navigate to the **Credits** tab to analyze how your company uses Cobalt [credits](https://www.cobalt.io/pentest-pricing#cobaltcredits).
+{{% /pageinfo %}}
+
+Here, you can see the following:
+
+- Current credit balance
+- {{% ptaas-tier %}} shown as "Subscription Plan"
```

Comment:
If the customer is in the "legacy" plan they will not see this Subscription Plan.

Reply:
Is it an "edge case"?

Reply:
@ana-dashuk-cobalt : not an edge case - 81% of our active orgs are on legacy. https://datastudio.google.com/u/0/reporting/1-w0oWYfNUx_YIjRlXn4oRU3OfMAQOOSn/page/p_ozg2iht4rc